app/routes/planet_routes.py

```diff
@@ -0,0 +1,94 @@
+from flask import Blueprint, abort, make_response, request, Response
+from app.routes.db import db
+from app.models.planet import Planet
+
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+
+@planets_bp.post("")
+def create_planet():
+
+    request_body = request.get_json()
+    name = request_body["name"]
+    description = request_body["description"]
+    distance_from_sun = request_body["distance_from_sun"]
+
+    new_planet = Planet(
+        name=name, description=description, distance_from_sun=distance_from_sun
+    )
+    db.session.add(new_planet)
+    db.session.commit()
+
+    response = new_planet.to_dict()
+
+    return response, 201
```

Comment:
We could move all of this logic into a function like `create_model` that we created in class.